### PR TITLE
Standarize Readme

### PR DESCRIPTION
**Pull request by RichardLitt**
See https://github.com/ipfs/community/issues/124

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 IPFS
```
Comment (whyrusleeping) [inline LICENSE]: ipfs isnt an entity so we cant have copyrights attributed to it

**Comment by whyrusleeping** [comment]
comment re: license file.

Also, do you think you could help filling this repo out in general?

File: LICENSE (hunk 1 above)
Comment (RichardLitt) [inline LICENSE]: Fair. It's actually attributed in a lot of these. Let's discuss in https://github.com/ipfs/community/issues/139.

**Comment by RichardLitt** [comment]
I can! Yeah. Happy to take control and add more things?

**Comment by RichardLitt** [comment]
You're right, of course; removed License for now. We should switch to CC-BY-SA, but no rush. Will open issue for that.

**Comment by dignifiedquire** [comment]
LGTM

**Pull request merged**